ccm_web/client/src/pages/MergeSections.tsx

```diff
@@ -294,6 +294,16 @@ function MergeSections (props: CCMComponentProps): JSX.Element {
     <Root className={classes.root}>
       <Help baseHelpURL={props.globals.baseHelpURL} helpURLEnding={props.helpURLEnding} />
       <Typography variant='h5' component='h1' className={classes.spacing}>{props.title}</Typography>
+      <div>
+        <Typography paragraph>
+          <strong>Caution!</strong> If you merge a section from a course that contains content, 
+           submissions and grades, that content will <i>stay with the original course </i>
+           once the section is merged, becoming inaccessible to teachers and students. 
+        </Typography>
+        <Typography paragraph>
+          It is recommended to merge sections <i>before</i> the start of term.
+        </Typography>
+      </div>
```

Comment:
I personally feel the `It is recommended to merge sections <i>before</i> the start of term.` could be in the same paragraph above, since that pushes the view for merging section little down. Thoughts @ktowneUM 

<img width="1257" height="135" alt="Image" src="https://github.com/user-attachments/assets/c28b7757-abac-498e-84c0-90606ad1e5f4" />